src/components/Footer.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import * as style from "../styles/homestyle";
+
+const Footer = props => {
+  const seeFriends = e => {
```

Comment:
Wonder what this method is for. It is a good practice to add comment to explain what you're trying to accomplish in a method.